### DISCUSSION TITLE
Is it possible to pass variables in to the ticket templates?

### DISCUSSION THREAD

**Discussion by jalfaro-do** [ankitpokhrel/jira-cli#672] [Q&A]
I'm generating ticket templates for use with jira-cli, but haven't found any documentation around being able to pass variables into the ticket template - if I can't do that, then I suppose I forego the file template and just build my templates in bash where I can substitute in the variables and build the jira command that way.

**Comment by ankitpokhrel** [comment]
Hi @jalfaro-do, variables are not supported on the template. Could you please explain the usecase to understand this better?

**Reply by kdwitt** [reply]
I can't speak for the OP, My use case would be:

I have an epic for patching of all systems in my datacenter, I need to create tasks under the epic for 100+ groups of systems. I'd like to use `jira issue create ` with a template in a for loop, where i can assign the variable from the for statement to a variable in the description template.
